metaserver/src/test/scala/tests/compiler/CompilerSuite.scala

```diff
@@ -0,0 +1,69 @@
+package tests.compiler
+
+import scala.meta.languageserver.Compiler
+import scala.reflect.internal.util.Position
+import scala.tools.nsc.interactive.Global
+import tests.MegaSuite
+
+class CompilerSuite(implicit file: sourcecode.File) extends MegaSuite {
+  val compiler: Global =
+    Compiler.newCompiler(classpath = "", scalacOptions = Nil)
+
+  def pretty(position: Position): String =
+    s"""${position.lineContent}
+       |${position.lineCaret}""".stripMargin
+  val TIMEOUT = 100
+  private def computeDatabaseAndNamesFromMarkup(
+      filename: String,
+      markup: String
+  ): List[Position] = {
+    val chevrons = "<<(.*?)>>".r
+    val carets0 =
+      chevrons.findAllIn(markup).matchData.map(m => (m.start, m.end)).toList
+    val carets = carets0.zipWithIndex.map {
+      case ((s, e), i) => (s - 4 * i, e - 4 * i - 4)
+    }
+    val code = chevrons.replaceAllIn(markup, "$1")
+    val unit = Compiler.addCompilationUnit(compiler, code, filename)
+    carets.map {
+      case (start, end) =>
+        unit.position(start)
+    }
+  }
+
+  trait OverloadHack1; implicit object OverloadHack1 extends OverloadHack1
+  trait OverloadHack2; implicit object OverloadHack2 extends OverloadHack2
+  trait OverloadHack3; implicit object OverloadHack3 extends OverloadHack3
+  trait OverloadHack4; implicit object OverloadHack4 extends OverloadHack4
+  trait OverloadHack5; implicit object OverloadHack5 extends OverloadHack5
```

Comment:
What's this for? If it's for disambiguating the erased signature, can you maybe use DummyImplicit on just one of the targeted signature below?

Reply:
This is copied from the scalameta test suite where there are 6 `targeted` overloads. It seems we don't need any overloads here so we can just remove this.